### PR TITLE
HTML/CSS & JavaScript capstone project-Online Shopping Page

### PR DESCRIPTION
**Pull request by Lawmsangi**
In this project, I implemented the following:

- The home page have a link in the menu to the About page.
- The logo in the header links to the home page.

> Mobile menu

- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.

> Dynamic page
- The section "Main Products" is created dynamically in JavaScript.
- I use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @Lawmsangi , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project with a readme having instructions. Please delete all instructions from your readme

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        </ul>
+    </div>
+     <div class="header-black">
+            <ul>
+                <li><a href=""><img src="images/facebook-64.png" alt="fb"></a></li>
+                <li><a href=""><img src="images/twitter-64.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">MyPage</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
```
Comment (V-Blaze) [inline index.html:30]: - [ ] You have done great so far :+1: :+1:  As part of the requirement for this project you are required to replicate the given design as much as you can. I think the height of this top navigation bar can be reduced to match the given design. looking at the screenshot below, you can see the difference in your implementation. I'll suggest you adjust this. :+1: :100: 

- - Your design:
Screenshot from 2023-06-28 09-57-57

- - Given design:
Screenshot from 2023-06-28 09-57-42

File: script.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        <img src="images/2.jpeg" alt="product-image">
+      </div>
+    <div class="product-details">
+        <h4>Bubu Dudu King Hoodie</h4>
+        <span>&#8377 999</span> &#8377 799
+        <hr class="line-black">
+        <img src="images/truck.svg" alt="truck" style="color:grey">
+        Standard shipping in 10 working days
+    </div>`;
+
```
Comment (V-Blaze) [inline script.js]: - [ ] Looking at the project requirement, I think you missed the requirement for this section. 
- - As stated in the project requirement, you are meant to have an array of objects here containing the data for your `speaker or Main Products` section. eg: `const speakerList = [ { name: "Speaker Name", image: './speaker1.png', ... }, { name: "Speaker Name", image: './speaker1.png', ... } ]`. then you can loop through the array using `speakerList.map() or speakerList.forEach()` generating HTML content for each of the speaker or in your case product and then append it to the page. Implementing it this way makes it easier to add a new item. I'll suggest you update this. :smile: :wave:

File: about.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                    <div class="partner-line"></div>
+            </div>
+            <div class="partner-content">
+                <img src="images/amazon.png" alt="amazon">
+                <img src="images/hm-logo.png" alt="h&m">
+                <img src="images/adidas.png" alt="adidas">
+                <img src="images/shopify.png" alt="shopify">
+                <img src="images/flipkart.png" alt="flipkart">
+            </div>
+        </div>
```
Comment (V-Blaze) [inline about.html]: - [ ] You have done great creating the partner section and styling it according to the Given design. :+1: :+1: 
- -  Looking at this from the user's perspective, the logos are not visible here, these are black logos in a dark background and therefore make it hard to see, I'll suggest you modify the color of these logos to ones similar in the design. :smile: :+1:  

- - Your current design:
Screenshot from 2023-06-28 10-12-22

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Lawmsangi , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Mobile menu works great✔
- Clean organized code👋 
-  UI looks great✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                    <div class="partner-line"></div>
+            </div>
+            <div class="partner-content">
+                <img src="images/amazon-64.png" alt="amazon">
+                <img src="images/nike-2-64.png" alt="nike">
+                <img src="images/apple-64.png" alt="apple">
+                <img src="images/google-plus-3-64.png" alt="google">
+                <img src="images/facebook-2-64.png" alt="fb">
+            </div>
+        </div>
```
Comment (usorfaitheloho) [inline about.html:89]: You did well implementing some of the changes mentioned by the previous reviewer 😄 but there seems to be just a minor issue to be addressed. For the partner's section of the `about page`, as suggested by the previous reviewer, you should try to make it similar to the design given by using the right background color for this section , it is always good to replicate a design as much as possible for a great UI👍

Behance design : 
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @Lawmsangi  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good use of JavaScript, HTML&CSS best practices ✔️ 
-  Well-structured files ✔️
- Very Good design✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Lawmsangi 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**